### PR TITLE
Extract report parsing to separate class

### PR DESCRIPTION
**Pull request by isimluk**

### CODE REVIEW

File: cs_scanimage.py [hunk 1]
```diff
@@ -118,20 +106,36 @@ def get_scanreport(self, token):
...
                 print("report not generated yet, retrying ... ")
             else:
-                return resp.json()
+                return ScanReport(resp.json())
         print("retries exhausted")
         raise APIError('GET ' + get_url + ' {}'.format(resp.status_code))
 
+
+class ScanReport(dict):
+    """Summary Report of the Image Scap"""
```
Comment (redhatrises) [inline cs_scanimage.py:115]: @isimluk I assume that you mean `Scan` instead of `Scap`?

Comment (isimluk) [reply cs_scanimage.py:115]: muscle memory, you know...

**Comment by redhatrises** [comment]
LGTM. Thanks for this!

File: cs_scanimage.py (hunk 1 above)
Comment (shawndwells) [reply cs_scanimage.py]: lol

**Comment by shawndwells** [comment]
+cc @ashwinathalye-cs for awareness

**Pull request merged**